ocean_provider/utils/provider_fees.py

```diff
@@ -77,7 +77,7 @@ def get_provider_fees(
         "v": (signed.v + 27) if signed.v <= 1 else signed.v,
         "r": web3.toHex(web3.toBytes(signed.r).rjust(32, b"\0")),
         "s": web3.toHex(web3.toBytes(signed.s).rjust(32, b"\0")),
-        "validUntil": valid_until,
+        "duration": duration,
```

Comment:
for provider fee response? Honestly, I disagree. We have seen a huge problem in partially switching from serviceIndex to serviceId and we ended up deciding to only keep the old semantics when dealing with the smart contracts. I think we should use the same reasoning here, by keeping validUntil only in dealing with the smart contracts (and renaming it in smart contracts as soon as possible, when other changes appear)

Reply:
Entire providerFee object has two flows:
 - the lib should pass it through to the contracts
 - if feeAmount>0, then the lib should pay that fee
 By using 'duration' instead of 'validUntil' means that in the lib we have to duplicate the object, change one key (duration -> validUntil) which leads to a lot of confusion.  Also, for devs using something else then ocean.js/ocean.py, this becomes a real hurdle. 
So let's use 'validUntil'